src/static/support/dist-docs/ovn-appctl.8.html

```diff
@@ -1,8 +1,10 @@
 <html><head><meta charset="UTF-8"></head><body><pre>
-ovn-appctl(8)                 Open vSwitch Manual                ovn-appctl(8)
+ovn-appctl(8)                     OVN Manual                     ovn-appctl(8)
 
 
 
+build/.PP
```

Comment:
typo?

Reply:
hm... good catch. I think this is an issue with the dist-docs build in the core [ovn repo](https://github.com/ovn-org/ovn/blob/master/Documentation/ref/index.rst). I am particular curious about the fact that only a few of them have that.
I will have to play a little more with that in order to understand why some of the generated files
have that weird blurb on them. I wonder if @blp or @putnopvut would already know.